freud/test_Hexatic.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2019 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+
+import freud
+from ovito.data import *
+
+def sense_check(k):
+    if k==0: print('WARNING! OVERWRITE k VALUE IN SCRIPT')
+    return
```

Comment:
No return needed, also instead of printing you can probably just raise an error. Alternatively, we could just have a convention of using default values in this repo that will cause freud to error. That would require a separate PR to freud that improves error checking, but I think that's probably a good thing to do in freud anyway (i.e. make sure that k>0 for hexatic). @bdice thoughts?

Reply:
OVITO will display that printed messages in a user-friendly way. I don't know if raising an error is as easy to decipher.

Reply:
Errors raised in OVITO show up in the output section of the script editor when it's open, or in the output window in the modifier section when it's not open. It will also have the usual traceback, which helps pinpoint the error.